server/games/game.py

```diff
@@ -669,19 +669,24 @@ async def update_game_stats(self):
             )
             row = await result.fetchone()
 
-            if row:
-                self.map_id = row['id']
+        is_generated = (self.map_file_path and "neroxis_map_generator" in self.map_file_path)
 
-            if (not row or not row['ranked']) and self.validity is ValidityState.VALID:
-                await self.mark_invalid(ValidityState.BAD_MAP)
+        if row:
+            self.map_id = row['id']
 
-            modId = self.game_service.featured_mods[self.game_mode].id
+        if (self.validity is ValidityState.VALID and
```

Comment:
I like binary operators at the start of the new line rather than the end of the last (and I think we are doing that somewhat consistently?), like
```python
if (
  self.validity is ValidityState.VALID
  and ((row and not row.ranked) or (not row and not is_generated))
):
  await  self.mark_invalid(ValidityState.BAD_MAP)
```
but that's of course only a minor thing.

Reply:
Consistent style is important noted